css/base.css

```diff
@@ -48,22 +48,28 @@ table {
 	border-collapse: collapse;
 	border-spacing: 0;
 }
```

Comment:
Your reset should be in a separate `reset.css` file, rather than in your `base.css`.